packages/make-styles/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@fluentui/make-styles",
+  "version": "0.1.0",
+  "description": "---",
+  "private": true,
```

Comment:
I understand we agreed to make this package private but this makes it non-showable in Northstar docsite :-(

Reply:
Made it public in 828acf6. Agree, we discussed that yesterday and it makes hard to test it on consumers' environments